src/elements/ExpiryIndicator.stories.ts

```diff
@@ -0,0 +1,45 @@
+import {fn} from '@storybook/test';
```

Comment:
Unused. Can we remove this?

Reply:
I saw these unused in other files too so I'm not entirely sure.